cupy/core/core.pxd

```diff
@@ -15,6 +15,8 @@ cdef class ndarray:
         public strides_t _strides
         readonly bint _c_contiguous
         readonly bint _f_contiguous
+        # To do fast indexing in the CArray class
+        readonly bint _index_32_bits
```

Comment:
This is to avoid accessing python runtime when creating the ArgInfo's for the kernel, otherwise this greatly affects CPU time on kernel launch.